js_modules/dagster-ui/packages/ui-core/src/app/HelpMenu.tsx

```diff
@@ -127,6 +148,32 @@ export const HelpMenuContents = ({
           text="Contact sales"
         />
       ) : null}
+      <MenuDivider title="Version" />
+      <Box
+        flex={{direction: 'row', gap: 8, alignItems: 'center', justifyContent: 'space-between'}}
+        padding={{vertical: 4, horizontal: 8}}
+      >
+        <div style={{fontSize: '12px', color: Colors.textLight()}}>
+          {version ? (
+            <span style={{fontFamily: FontFamily.monospace}}>{version}</span>
+          ) : (
+            <Spinner purpose="caption-text" />
+          )}
+        </div>
+        <Tooltip
```

Comment:
I wonder if this could re-use `<CopyIconButton />` ? At one point I was trying to burn down these custom copy components